src/store/initial-state.js

```diff
@@ -24,6 +24,7 @@ export const createInitialState = () => ({
   expandAllPipelines: false,
   isPrettyName: settings.isPrettyName.default,
   showFeatureHints: settings.showFeatureHints.default,
+  showDatasetPreviews: settings.showDatasetPreviews.default,
```

Comment:
It is good to have a nesting, but how are you able to nest if the initial state is at the same level as flags ?
https://github.com/kedro-org/kedro-viz/pull/1977/files#r1688315574

Reply:
Fair you are right, we can go back to original.